pkg/apis/metalkube/v1alpha1/baremetalhost_types.go

```diff
@@ -90,12 +90,17 @@ type CPU struct {
 
 // Storage describes one storage device (disk, SSD, etc.) on the host.
 type Storage struct {
-	SizeGiB int    `json:"sizeGiB"`
-	Info    string `json:"info"`
+	Name      string `json:"name"`
+	Interface string `json:"interface"`
+	SizeGiB   int    `json:"sizeGiB"`
+	Class     string `json:"class"`
```

Comment:
What is "class"? Where will that come from? Is it derived from information we can see when we inspect the hardware on the host?

Reply:
Storage class; it seemed redundant to call it `Storage.StorageClass`.

Reply:
Sure, the name is fine. I just don't know where the value comes from. :-)